packages/@sfx/sayt-driver-plugin/CHANGELOG.md

```diff
@@ -8,3 +8,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - SFX-160: Added the `SaytDriverPlugin` module.
   - The `SaytDriverPlugin` module acts as a mediator between the Logic and View layers for prebuilt modules.
+
+- SFX-295: Added the origin searchbox ID to the `saytResponseEvent`
```

Comment:
Since this is still unreleased, I would add this as a sub-point under SFX-160. Like so:

```markdown
- SFX-160: Added the `SaytDriverPlugin` module.
  - The `SaytDriverPlugin` module acts as a mediator between the Logic and View layers for prebuilt modules.
  - SFX-295: Added the associated searchbox ID to response events.
```